csrankings-r.csv

```diff
@@ -1385,7 +1384,7 @@ Ryan Kok Leong Ko,University of Queensland,https://researchers.uq.edu.au/researc
 Ryan Layer,University of Colorado Boulder,http://layerlab.org,AXnmFTIAAAAJ
 Ryan M. Layer,University of Colorado Boulder,http://layerlab.org,AXnmFTIAAAAJ
 Ryan Marcus,University of Pennsylvania,https://rmarcus.info/,vPOl-IwAAAAJ
-Ryan Newton, Purdue University, https://www.cs.purdue.edu/people/faculty/rrnewton.html, Bp6dm-oAAAAJ
+Ryan Newton,Purdue University,https://www.cs.purdue.edu/people/faculty/rrnewton.html,Bp6dm-oAAAAJ

```

Comment:
Ryan's LinkedIn does not indicate he is full-time at Purdue. Please remove.
https://www.linkedin.com/in/rrnewton/